app/src/androidTest/java/org/mozilla/fenix/ui/robots/HomeScreenRobot.kt

```diff
@@ -673,6 +674,14 @@ private fun verifySearchEngineIcon(searchEngineName: String) {
     verifySearchEngineIcon(ddgSearchEngine.icon, ddgSearchEngine.name)
 }
 
+private fun getNimbus() =
+    appContext.components.analytics.experiments
```

Comment:
This isn't reused elsewhere, I would prefer to see it inlined in the one place it is being used.